Help.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/* 
+ * File:   Help.h
+ * Author: Feriel
+ *
+ * Created on 13 décembre 2020, 12:41
+ */
+
+#ifndef HELP_H
+#define HELP_H
+
+#include <iostream>
+#include <string>
+
+
+class Help

```

Comment:
This class rather be simplistic. It can simply store different help messages as public static constants. It should also be not constructible (PRIVATE CONSTRUCTOR); in other words, there is no use of having an instance of Help.